app/routes/circulars/circulars.server.ts

```diff
@@ -539,7 +539,8 @@ export async function approveChangeRequest(
   })
 
   await deleteChangeRequestRaw(circularId, requestorSub)
-
+  if (changeRequest.zendeskTicketId)
+    await closeZendeskTicket(changeRequest.zendeskTicketId)
```

Comment:
The only reason that we would need this is to handle change requests that pre-date the deployment of this PR, right?
```suggestion
    await closeZendeskTicket(changeRequest.zendeskTicketId)
```

Also, can we do some of these steps concurrently (`Promise.all`)?

Reply:
Yup, and I can check on the concurrent thing. I assume probably